{{ cookiecutter.project_name_dashed }}/.github/ISSUE_TEMPLATE/general.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: 🔧 General Issue
+description: Create a general issue report
+labels: ["triage"]
+body:
+  - type: dropdown
+    id: type
+    attributes:
+      label: Issue Type
+      description: What type of issue is this?
+      options:
+        - Bug Report
+        - Feature Request
+        - Documentation
+        - Question
+        - Other
+    validations:
+      required: true
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Clear and concise description of the issue
+      placeholder: |
+        For bugs: What happened? What did you expect to happen?
+        For features: What need are you trying to address?
+    validations:
+      required: true
+
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Steps to Reproduce
+      description: Clear steps to reproduce the behavior (if applicable)
+      placeholder: |
+        1. Go to '...'
+        2. Click on '...'
+        3. See error
+    validations:
+      required: false
+
+  - type: textarea
+    id: solution
+    attributes:
+      label: Proposed Solution
+      description: Your ideas for implementing the feature or fixing the bug
+      placeholder: I think this could be solved by...
+    validations:
+      required: false
+
+  - type: textarea
+    id: context
+    attributes:
+      label: Additional Context
+      description: Any other context, screenshots, or relevant information
+      placeholder: |
+        - Environment details
+        - Related issues
+        - Links to documentation
+    validations:
+      required: false
+
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Pre-submission Checklist
+      options:
+        - label: I have searched existing issues
+          required: true
+        - label: I have read the documentation
```

Comment:
Maybe use consistent wording: "I have read the documentation" / "I have gone through the documentation" etc.